stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -52,6 +52,13 @@ internal class PaymentOptionsAdapter(
         notifyDataSetChanged()
     }
 
+    internal fun setEnabled(enabled: Boolean) {
+        if (isEnabled != enabled) {
+            isEnabled = enabled
+            notifyDataSetChanged()
+        }
+    }
```

Comment:
Totally optional, but you could combine this and the property definition into one like this:

```
    internal var isEnabled: Boolean by Delegates.observable(
        true
    ) { _, oldValue, newValue ->
        if (oldValue != newValue) {
            notifyDataSetChanged()
        }
    }
```

Reply:
Done